packages/zengine-ui-react/src/util/getStates.ts

```diff
@@ -0,0 +1,212 @@
+import { SelectOption } from '../atoms/Select';
+
+const getStates = (): SelectOption[] => {
```

Comment:
im wary about hard-coding states and countries... We recently went through some country changes and it was a pain to need to update in several places ... bulk update, wigeon, sandy. Can we pull this from the zengine context?
Also... the state field has this somewhat strange configuration where if u "include provinces/territories", then it include canadian provinces and U.S. territories. This state field just has the 50 states.